analyzers/integration_tests/testtools/testtools.go

```diff
@@ -0,0 +1,69 @@
+package testtools
+
+import (
+	"github.com/fossas/fossa-cli/exec"
+
+	git "gopkg.in/src-d/go-git.v4"
+	"gopkg.in/src-d/go-git.v4/plumbing"
+)
+
+type Initializer interface {
```

Comment:
I don't think using an interface is the right level of abstraction. This seems too general to me.

The only consumer of this interface is `SetupTests`, which just calls `BuildAll` and `RemoveAll`.